sbc_translation/models/correspondence.py

```diff
@@ -22,22 +34,114 @@ class Correspondence(models.Model):
     ##########################################################################
     #                              ORM METHODS                               #
     ##########################################################################
+
     @api.model
     def create(self, vals):
-        """ Create a message for sending the CommKit only if the preferred
-            language is the same as the letter's language else send the letter
-            on the local translate plaforme.
-            """
-
+        """ Create a message for sending the CommKit after be translated on
+             the local translate plaforme.
+        """
         sponsorship = self.env['recurring.contract'].browse(
             vals['sponsorship_id'])
-        if sponsorship.reading_language.id == vals['original_language_id']:
-            letter = super(Correspondence, self).create(vals)
-        else:
+
+        original_lang = self.env['res.lang.compassion'].browse(
+            vals['original_language_id'])
+
+        if original_lang.translatable and original_lang.id not in sponsorship\
+                .child_id.project_id.country_id.spoken_lang_ids.mapped('id'):
```

Comment:
You can simply test `... and original_lang not in sponsorship.child_id.project_id.country_id.spoken_lang_ids`